### PR TITLE
[#76] TableDocElement shouldn't render empty header

### PR DESCRIPTION
**Pull request by nils-christian**
Signed-off-by: Nils Christian Ehmke <nils@rhocas.de>

### CODE REVIEW

File: integration/asciidoc/src/main/java/org/archifacts/integration/asciidoc/TableDocElement.java [hunk 1]
```diff
@@ -65,14 +65,21 @@ private void renderStartEndTag(final StringBuilder stringBuilder) {
 	}
 
 	private void renderHeader(final StringBuilder stringBuilder) {
-		for (final TableDocElementColumn<T> column : columns) {
-			stringBuilder.append('|');
-			if (column.title != null) {
-				stringBuilder.append(column.title);
+		final boolean atLeastOneColumnTitleIsNotNull = columns
+				.stream()
+				.map(c -> c.title)
```
Comment (OLibutzki) [inline integration/asciidoc/src/main/java/org/archifacts/integration/asciidoc/TableDocElement.java:70]: In TableDocElementTest you introduced getters in order to use method refeences. I would prefer to use this approach here as well --> TableDocElementColumn::getTitle

**Review by OLibutzki — CHANGES_REQUESTED** [review]
Two minor notes... you can leave it as is if your prefer to...

File: integration/asciidoc/src/main/java/org/archifacts/integration/asciidoc/TableDocElement.java (hunk 1 above)
Comment (nils-christian) [reply integration/asciidoc/src/main/java/org/archifacts/integration/asciidoc/TableDocElement.java]: I added further (private) getters to access the fields.

**Comment by nils-christian** [comment]
Two notes? I can see only one (about the getters for method references).

**Comment by OLibutzki** [comment]
The second one was about `Objects::nonNull`. As you "fixed" that I will merge this PR.

**Pull request merged**